### PR TITLE
order Added

### PR DESCRIPTION
**Pull request by ArabindaAutomation**
Hi Arabinda2018

Can you please check this pr?

* Added order page
* Added order test cases

Thanks
ArabindaAutomation

### CODE REVIEW

File: src/pages/Order.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package pages;
+
+public class Order {
+
+	public static void main(String[] args) {
```
Comment (arabinda2018) [inline src/pages/Order.java:5]: Where is order page action method.

Comment (arabinda2018) [inline src/pages/Order.java:5]: Where is the logic

File: src/tests/OrderTest.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package tests;
+
+public class OrderTest {
+public void m1() {
```
Comment (arabinda2018) [inline src/tests/OrderTest.java]: Where is @Test annotation ?

Comment (arabinda2018) [inline src/tests/OrderTest.java]: Where is assertion logic

**Review by arabinda2018 — CHANGES_REQUESTED** [review]
Hi,
Please check my review comments an make the changes accordingly 

arabinda 2018

File: src/pages/Order.java (hunk 1 above)
Comment (ArabindaAutomation) [reply src/pages/Order.java:5]: done

Comment (ArabindaAutomation) [reply src/pages/Order.java:5]: added

**Review by arabinda2018 — APPROVED** [review]

**Comment by ArabindaAutomation** [comment]
Order page changes to merging to the master branch

**Pull request merged**